index.html

```diff
@@ -450,9 +475,8 @@
         <a>PushManager</a> interface
       </h2>
       <p>
-        The <a>PushManager</a> interface defines the operations that enable <a title=
-        "webapp">webapps</a> to establish access to <a title="push service">push services</a>. Note
-        that just a single <a>push subscription</a> is allowed per <a>webapp</a>.
```

Comment:
This was an inconsistency in the spec and its removal is intentional. I've updated the change description to clarify.